src/lib.rs

```diff
@@ -208,6 +222,18 @@ impl<'a> Ammonia<'a> {
             child.parent.replace(None);
             dom.append(&parent.clone(), NodeOrText::AppendNode(child.clone()));
         }
+        pass
+    }
+
+    fn fix_child(&self, child: &mut Handle, link_rel: &Option<StrTendril>) {
```

Comment:
I guess it makes more sense to check that the config is valid in `clean` than in `fix_child`...